themes/stanford_basic/stanford_basic.theme

```diff
@@ -226,6 +226,10 @@ function stanford_basic_preprocess_block(&$variables) {
       unset($variables['content']);
     }
   }
+
+  if (!isset($variables['configuration']['label_display'])) {
+    $variables['attributes']['aria-hidden'] = 'true';
```

Comment:
How do I get it applied to the `h2` instead? I haven't figured that part out.  Also is this what you were thinking?

Reply:
By looking at the template, you can see that it uses the `title_attributes` variable: https://github.com/SU-SWS/stanford_profile/blob/8a78aba443667c54ccf3189211b98079f6345c4b/themes/stanford_basic/templates/block/block--system-menu-block.html.twig#L50